Maths/Fibonacci.ts

```diff
@@ -0,0 +1,34 @@
+/**
+ * A function to get Fibonacci value of given number
+ * @param number The input integer
+ * @return {number} Fibonacci value of `number`
+ * @example fibonacciValue(4) => 3 | fibonacciValue(6) => 8
+ * @see https://en.m.wikipedia.org/wiki/Fibonacci_number
+ * @author MohdFaisalBidda <https://github.com/MohdFaisalBidda>
+ */
+
+export const fibonacciValue = (number: number): number => {
+  if (typeof number === "string") {
+    throw "Input should be a number";
```

Comment:
The `number` parameter type annotation above should already take care of this, making this check redundant.